src/main/java/io/quarkus/gizmo/Signature.java

```diff
@@ -0,0 +1,49 @@
+package io.quarkus.gizmo;
+
+public interface Signature {
```

Comment:
I don't see a reason for this interface. Also for every signature you'll create a completely useless object (e.g. `ClassSignature`) and it's also more difficult to use: `Signature.forClass().build().toString()` VS `SignatureBuilder.forClass().build()`...

Reply:
Ugh. I don't recall adding this, I probably had this locally from my previous unfinished experiments with this. It doesn't make any sense, it's the same as `SignatureBuilder`. Will delete.

Reply:
Done.